bucket/nuwen-gcc.json

```
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://nuwen.net/mingw.html",
+    "description": "Nuwen's MinGW Distro with GCC, Boost etc.",
+    "license": "GPL-2.0-only",
+    "version": "11.2.0-18.0",
```

```suggestion
    "version": "18.0",
```

18.0 is the version of the distro - otherwise you'll have to write the version of everything (the MinGW version is actually 9.00)